cli/cmd/resource_run_cmd.go

```diff
@@ -102,3 +121,20 @@ func validRequiredGatesMsg() string {
 
 	return "valid options: " + strings.Join(opts, ", ")
 }
+
+func hasValidResourceFiles() bool {
+	if len(runParams.DefinitionFiles) == 0 {
```

Comment:
Should an empty list be valid?

Reply:
Yes, in the case of the OSS test. (We can use --id for that case.)
Perhaps we should have something to validate when using run groups, too?